Cql/Cql.Packaging/CqlCrosswalk.cs

```diff
@@ -13,9 +13,9 @@
 using Hl7.Fhir.Model;
 namespace Hl7.Cql.Packaging
 {
-    internal class CqlCrosswalk
+    public class CqlCrosswalk

```

Comment:
Can you remove the

`#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member`

from the top of the file and document this class?

Reply:
Sure,  I will update it in a subsequent PR.